### ISSUE TITLE
Investigation: expanding the BAR query

### ISSUE THREAD

**Issue by Maribeth-Wilson** [bcgov/business-ar#475]
understanding the query and what corps we could include; ie. what resides in LEAR

**Comment by Maribeth-Wilson** [comment]
@hanlunBCRegistries - do you have any updates?

**Comment by hanlunBCRegistries** [comment]
@Maribeth-Wilson Yes, currently we included basically all corp types except Benefit companies here is current criteria:

Corporation Type and Class:
Corporation class must be 'BC' (British Columbia Corporations).
Corporation type must not be 'BEN' (Benefit Companies).

Email and AR Reminder:
The corporation must have a non-null admin email.
The AR (Annual Report) reminder indicator must be 'Y'.

Active Status:
The corporation state must be 'ACT' (active).
The corporation name type must be 'CO'.
The corporation state and name must not have an end event.

No Previous BCOL Filings:
The corporation must not have any previous BCOL (BC Online) filings.

Unique Email:
The admin email must not be used by any other corporation.

No SBC Connect Account:
The admin email must not be associated with any SBC Connect account.
The business identifier must not already exist in the Auth-db.

Recognition Date:
The recognition date plus the number of years since recognition plus one day must equal the current date.

Filing Date:
The corporation must have been founded in the previous year and not have filed an AR.
Or, if the last AR filed date is not null, it must have been filed in a previous year.

**Comment by Maribeth-Wilson** [comment]
Great, thank you. @hanlunBCRegistries what else is needed?

**Comment by hanlunBCRegistries** [comment]
@Maribeth-Wilson Just need to wait for list of mass affiliation companies and to exclude those companies and then we are finished with extending BAR I believe.

**Comment by Maribeth-Wilson** [comment]
closing this tix. Same details in #508
